### PR TITLE
Added an example of mixed format argv in command module

### LINKED ISSUE #80126: More detailed description about how command module's argv parameters works
### Summary

Writing this following #79967, which I created because I thought that I encountered a bug, but in fact I didn't know some details about how `argv` parameter works.

It would be great to have information provided to me there in the documentation in case anybody else encounter the same issues. Especially this comment was very helpful:
> precisely because you can use white space you need to separate what is not the same argument or the flag + argument are passed as a unit '--flag arg' vs '--flag' 'arg'

### Issue Type

Documentation Report

### Component Name

https://github.com/ansible/ansible/blob/devel/lib/ansible/modules/command.py

### Ansible Version

```console
$ ansible --version
```

### Configuration

```console
# if using a version older than ansible-core 2.12 you should omit the '-t all'
$ ansible-config dump --only-changed -t all
```

### OS / Environment

Linux

### Additional Information

When the improvement is applied, it makes it more straightforward to understand how argv parameters works

### Code of Conduct

- [X] I agree to follow the Ansible Code of Conduct

### PR DESCRIPTION
**Pull request by Akasurde**
##### SUMMARY

Fixes: #80126

Signed-off-by: Abhijeet Kasurde <akasurde@redhat.com>

##### ISSUE TYPE
- Bugfix Pull Request

##### COMPONENT NAME
lib/ansible/modules/command.py

### CODE REVIEW

File: lib/ansible/modules/command.py [hunk 1]
```diff
@@ -161,6 +161,17 @@
...
     creates: /path/to/database
 
+- name: Run command using argv with mixed argument formats
+  ansible.builtin.command:
+    argv:
+      - /path/to/binary
+      - -v
+      - --debug
+      - --longopt
+      - value for longopt
```
Comment (oraNod) [inline lib/ansible/modules/command.py:171]:
```suggestion
      - value_for_longopt
```

File: lib/ansible/modules/command.py [hunk 2]
```diff
@@ -161,6 +161,17 @@
...
+      - --other-longopt=value for other longopt
```
Comment (oraNod) [inline lib/ansible/modules/command.py:172]:
```suggestion
      - --other-longopt=value_for_other_longopt
```

File: lib/ansible/modules/command.py (hunk 1 above)
Comment (sivel) [reply lib/ansible/modules/command.py:171]: This is specifically using spaces, to show that in argv an argument with spaces is considered to be a single argument, and not multiple arguments.

File: lib/ansible/modules/command.py (hunk 2 above)
Comment (sivel) [reply lib/ansible/modules/command.py:172]: The same comment here as the one above

**Review by jborean93 — APPROVED** [review]

File: lib/ansible/modules/command.py (hunk 1 above)
Comment (oraNod) [reply lib/ansible/modules/command.py:171]: TIL. Thanks for the explanation.

**Pull request merged**